labs/lab-4-jquery/table.js

```diff
@@ -0,0 +1,45 @@
+class Table {
+    constructor(tableNum, status) {
+        this.tableNum = tableNum;
+        this.status = status;
+    }
+
+    reserveTable(name, phoneNum, peopleNum) {
+        this.reserver = new Party(name, phoneNum, peopleNum);
+        this.status = "reserved";
+    }
+
+    openTable() {
+        this.status = "available";  
+        this.reserver = null;
+    }
+
+    render() {
+        var _this = this;
```

Comment:
Ah! You have learned a dangerous and powerful magick. Remind me to discuss _this in class tomorrow!